src/sql/platform/accounts/common/accountStore.ts

```diff
@@ -140,6 +149,15 @@ export default class AccountStore implements IAccountStore {
 			updatedAccounts: accounts
 		};
 	}
+	private removeAllAccountsList(): AccountListOperationResult {
+		return {
```

Comment:
Do you need to create this object? I don't see it used anywhere other than its `accountRemoved` field. You could just return true in that case.

Reply:
Huh I'm so confused why I did this lol.